static/src/js/components/Errors/NotFound.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import uuidv4 from 'uuid/v4'
+
+import LoggerRequest from '../../util/request/loggerRequest'
+
+const NotFound = ({ location }) => {
+  // eslint-disable-next-line global-require
+  require('./ErrorBoundary.scss')
+
+  const guid = uuidv4()
+
+  const requestObject = new LoggerRequest()
+  requestObject.log({
+    error: {
+      guid,
+      message: '404 error',
```

Comment:
I see location here, and stack some place else. Is there no expected payload here? We should try and keep things consistent (even if its by type of error)